.github/workflows/test-application.yaml

```diff
@@ -0,0 +1,80 @@
+name: Test application
+
+on:
+    pull_request:
+    push:
+        branches:
+            - master
+            - develop
+            - '[0-9]+.[0-9]+'
+
+jobs:
+    test:
+        name: "Run tests with php ${{ matrix.php-version }}"
+        runs-on: ubuntu-18.04
+
+        env:
+            SYMFONY__DATABASE__PASSWORD: root
+
+        strategy:
+            fail-fast: false
+            matrix:
+                include:
+                    - php-version: '5.5'
+                      composer-flags: '--prefer-lowest --prefer-dist --no-interaction'
+                      tools: 'composer:v1'
+                      env:
+                          SYMFONY__DEPRECATIONS__HELPER: disabled
+                    - php-version: '7.4'
+                      composer-flags: '--prefer-dist --no-interaction'
+                      tools: 'composer:v2'
+                      env:
+                          SYMFONY__DEPRECATIONS__HELPER: weak
```

Comment:
Are you sure this env variable should not be called `SYMFONY_DEPRECATIONS_HELPER`?